modules/plugins/notes/obsidian/obsidian.nix

```diff
@@ -45,10 +45,10 @@ in {
 
         completion = {
           nvim_cmp = mkOption {
-            # if using nvim-cmp, otherwise set to false
+            # If using nvim-cmp, otherwise set to false
             type = bool;
             description = "If using nvim-cmp, otherwise set to false";
-            default = config.vim.autocomplete.type == "nvim-cmp";
+            default = config.vim.autocomplete.nvim-cmp.enable;
```

Comment:
This should be an internal option, which is possibly also readOnly. I see no reason a user would want to override this, but if you do see one feel free to make it only internal.

Reply:
I'm not sure I agree with that, deviating from the raw setupOpts approach should be avoided where possible. Maybe the user wants to disable the completion features, and use the other features of the plugin, and if not, it doesn't matter.